apps/web/src/pages/settings/tabs/components/Security.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, colors, Switch, Text } from '../../../../design-system';
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Description } from '../InAppCenterCard';
+import { ColorScheme, useMantineTheme } from '@mantine/core';
+
+export const Security = () => {
+  const [isHmacEnabled, setIsHmacEnabled] = useState<boolean>(false);
+  const { colorScheme } = useMantineTheme();
+
+  function handlerSwitchChange() {
+    setIsHmacEnabled((prev) => !prev);
+  }
+
+  return (
+    <>
+      <Title>Security</Title>
+      <SubTitle weight="bold">Enable HMAC encryption</SubTitle>
+      <InlineDiv>
```

Comment:
@ainouzgali I think we have some sort of a "label" and "description" props to show this sorts of designs right?

Reply:
Yes. I think this might work here: [Input wrapper with our global input styles](https://github.com/novuhq/novu/blob/989fd2cffc59fb7f6dd2949ddeba2e190c4df4ac/apps/web/src/pages/settings/components/BrandingForm.tsx#L125)

Reply:
Will update it shortly, thanks for pointing me in the right direction! @ainouzgali